### DISCUSSION TITLE
Customize Dashboard

### DISCUSSION THREAD

**Discussion by Asil616** [babybuddy/babybuddy#642] [Ideas]
I was looking for ways to customize the dashboard to add cards that I am using (e.g. Pumping & Weight) and change the order of the cards to put the most frequently used cards first (particularly useful when accessing on the phone with a smaller screen). Is there a way to do this? I couldn't find anything on the docs.

**Comment by cdubz** [comment]
Unfortunately there is no way to customize right now. This is the oldest open issue #34 and something I do really want to get to, just haven't been able to put the time and effort in to it.

**Comment by nbanderso** [comment]
@cdubz How about just adding a tile for pumping?  I'm really struggling to sell Baby Buddy to my wife when she has to go through the top menu to get to pumping info.  

Help a sleep deprived brother out!

**Comment by cdubz** [comment]
@nbanderso could you open a new issue for that? And describe what you think would be the most valuable info for a dashboard pumping card?

**Comment by lenaxia** [comment]
@Asil616 i created a grafana dashboard here: https://github.com/babybuddy/babybuddy/discussions/607

If you run grafana and use the babybuddy postgres backend you can extend from what I have. 

The dashboard is mobile friendly and can be reordered as you need. 

If you have any familiarity with sql you can extend what I have. Also happy to help you if you get stuck.

**Comment by nbanderso** [comment]
@lenaxia Oh my god I need this!  By chance are you running Grafana through the Home Assistant addon?  

I use Grafana in Home Assistant (with InfluxDB as a data source) and I have no idea how to import a json file.  

Thanks!

**Comment by lenaxia** [comment]
I'm not running it in HA but it should be basically the same. Here's a writeup of how to import dashboards https://rudimartinsen.com/2020/08/06/grafana-importing-dashboards/

Just copy my Json and point it at the postgres db that has your babybuddy data.
